### PR TITLE
Create basic local development guidelines

### LINKED ISSUE #2: Write basic local development tutorial
This project should be easy to edit locally after installation from DEB package if `pip install -e` is applied. This should be documented in README.

### PR DESCRIPTION
**Pull request by icedevml**
resolve #2

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   ```
+   git clone https://github.com/CERT-Polska/drakvuf-sandbox.git
+   cd drakvuf-sandbox
+   ```
+2. Install local `drakcore` and `drakrun`.
+   ```
+   python3 -m venv venv
+   source venv/bin/activate
+   pip3 install --editable ./drakcore/
+   pip3 install --editable ./drakrun/
```
Comment (msm-code) [inline CONTRIBUTING.md:29]: :thinking: nice

Maybe a word or two on how to veify that installation worked?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Development system
+------------------
+
+Very first thing to consider is to setup and configure your local instance of DRAKVUF Sandbox. There are two basic options in that matter:
+
+* Develop on local machine: Install Debian Buster in [VMware Workstation 15 Player](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html).
+* Develop on a remote server: Just get some bare-metal or rent a dedicated server (e.g. [Kimsufi](https://www.kimsufi.com/us/en/servers.xml)) with Debian Buster.
+
+**Caution!** Your host machine must be an Intel processor with VT-x and EPT support, even when using VMware or other nested virtualization.
```
Comment (msm-code) [inline CONTRIBUTING.md:12]: Maybe some oneliner to check if the processor is ok?

I guess something like

```
cpuinfo | grep xyzvwy
```

**Comment by icedevml** [comment]
Wait... I've push forced the old version while doing rebase xD

File: CONTRIBUTING.md (hunk 2 above)
Comment (icedevml) [reply CONTRIBUTING.md:13]: This is already provided by drakvuf bundle:
https://github.com/tklengyel/drakvuf/blob/307420439af2f4caf92bcf80199d39ce22f2171e/package/postinst#L54

the user would get huge warning after installation of `drakvuf-bundle` DEB that the processor doesn't support EPT.

File: CONTRIBUTING.md (hunk 1 above)
Comment (icedevml) [reply CONTRIBUTING.md]: There is a section about that but I forgot to push it. Now it should be present.

File: CONTRIBUTING.md (hunk 2 above)
Comment (icedevml) [reply CONTRIBUTING.md]: hm but adding it to the docs is also pretty ok, I've added a note about checking for EPT support

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+1. Open `drakcore/drakcore/app.py`
+2. Add these lines before `def main()`:
+   ```python
+   @app.route("/hello-world")
+   def hello_world():
+       return 'hello'
+   ```
+3. Save the file and execute `systemctl restart drak-web`
+4. Navigate to http://localhost:6300/hello-world, your new subpage should appear.
```
Comment (msm-code) [inline CONTRIBUTING.md:66]:
```suggestion
4. Navigate to `http://localhost:6300/hello-world`, your new subpage should appear.
```

**Pull request merged**